electrum/gui/qml/qeapp.py

```diff
@@ -173,6 +183,42 @@ def bindIntent(self):
         except Exception as e:
             self.logger.error(f'unable to bind intent: {repr(e)}')
 
+    @pyqtSlot(str, result=bool)
+    def hasPermission(self, permissionFqcn: str) -> bool:
```

Comment:
what is "fqcn" short for? "fully qualified .. name"? is this a term from the android dev docs, or just something you came up with in the spirit of FQDN?

Reply:
it's a bit of a java-ism. fully qualified constant name. no objection to change.